### PR TITLE
Fixed excessive point polling

### PR DESCRIPTION
**Pull request by JWGmeligMeyling**
Now we only fetch points right after a login, vote or classification, or every 100 seconds

### CODE REVIEW

File: app/scripts/controllers/points/pointsController.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 angular.module('moodCatApp')
   .controller('PointsCtrl', ['PointsService', '$scope', '$interval', function(PointsService, $scope, $interval) {
 
+      var that = this;
```
Comment (GijsWeterings) [inline app/scripts/controllers/points/pointsController.js]: seriously tho, This is better than the .bind(this) cloned 20 times

File: app/scripts/controllers/points/pointsController.js [hunk 2]
```diff
@@ -12,16 +14,20 @@ angular.module('moodCatApp')
...
       this.fetchPoints = function() {
-          PointsService.getPoints().then((function(points) {
-              this.points = points;
-          }).bind(this));
+          PointsService.getPoints().then(function(points) {
+              that.points = points;
+          });
       };
 
       // Queries the API for a point update every second.
```
Comment (GijsWeterings) [inline app/scripts/controllers/points/pointsController.js]: you forgot to edit this comment

**Comment by GijsWeterings** [comment]
you forgot to fetch points for the classifygame

**Comment by JWGmeligMeyling** [comment]
To my knowledge the classifygame uses the same classification request from the service.

File: app/scripts/services/classify/classificationService.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
  angular.module('moodCatAudio')
-   .service('ClassificationService', ['moodcatBackend', '$log', 'soundCloudService', function ClassificationService(moodcatBackend, $log, soundCloudService) {
+   .service('ClassificationService', ['moodcatBackend', '$log', 'soundCloudService', '$rootScope', function ClassificationService(moodcatBackend, $log, soundCloudService, $rootScope) {
```
Comment (Azkle) [inline app/scripts/services/classify/classificationService.js]: Waarom 2x $rootsScope toevoegen?

**Comment by Azkle** [comment]
Have verified and Jan-Willem is right.

Looks good to me!

**Pull request merged**